docs/rendering/modelextensions/index.md

```diff
@@ -0,0 +1,16 @@
+Model Extensions
+================
+
+Forge provides several extensions to the model JSON format to allow fine tuning the way JSON models are loaded, baked and rendered:
+
+- [Custom Model Loaders][modelloaders]
+- [Root Transforms][transforms]
+- [Render Types][rendertypes]
+- [Part Visibility][visibility]
+- [Face Data][facedata]
+
+[modelloaders]: ../modelloaders/index.md
+[transforms]: ./transforms.md
+[rendertypes]: ./rendertypes.md
+[visibility]: ./visibility.md
+[facedata]: ./facedata.md
```

Comment:
I'm not satisfied with the introduction section because, yes, it's basically empty. But for some reason it felt weird to not have an introduction with a short overview. I'll have to find a way to meaningfully expand on this.